trusted-types/eval-csp-tt-default-policy.html

```diff
@@ -8,7 +8,16 @@
 </head>
 <body>
 <script>
-  trustedTypes.createPolicy("default", {createScript: s => s});
+  trustedTypes.createPolicy("default", {
+    createScript: ((s, _, sink) => {
+      if (s === "1+1") {
```

Comment:
I think this is not super obvious, why this is working. Maybe do a simple regexp to check whether s contains an anonymous function prefix in order to decide it's a Function (and an eval otherwise)? Or at least comment that eval's sourceString will just have the evaluated text, while Function's sourceString would have a full function text.